tests/integration/golang/aim/experiment/delete_test.go

```diff
@@ -40,31 +38,14 @@ func (s *DeleteExperimentTestSuite) Test_Ok() {
 	defer func() {
 		assert.Nil(s.T(), s.fixtures.UnloadFixtures())
 	}()
-	experiment, err := s.fixtures.CreateExperiment(context.Background(), &models.Experiment{
-		Name: "Test Experiment",
-		Tags: []models.ExperimentTag{
-			{
-				Key:   "key1",
-				Value: "value1",
-			},
-		},
-		CreationTime: sql.NullInt64{
-			Int64: time.Now().UTC().UnixMilli(),
-			Valid: true,
-		},
-		LastUpdateTime: sql.NullInt64{
-			Int64: time.Now().UTC().UnixMilli(),
-			Valid: true,
-		},
-		LifecycleStage:   models.LifecycleStageActive,
-		ArtifactLocation: "/artifact/location",
-	})
+	test_experiments, err := s.fixtures.CreateExperiments(context.Background(), 1)
```

Comment:
can you please use `camelCase`?

Reply:
can we please create as minimum as possible test records for each test? so prev `s.fixtures.CreateExperiment` looks much better and understandable.